### PR TITLE
EPNG-5373: Make a blog post on 2019-06-20 about Let's Encrypt and Vault

### PR DESCRIPTION
**Pull request by ikoutras-epages**

### CODE REVIEW

File: _posts/2019/2019-06-20-managing-lets-encrypt-certificates-in-vault.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+about_authors: ["ikoutras"]
+---
+
+[Let's Encrypt](https://letsencrypt.org/){:target="_blank"} has facilitated the
+securing of websites by providing certificates free of charge. Here at ePages,
+we use them for testing our products, as well as for protecting some internal
+infrastructure. **Even though they are free, there is an extra cost: they need
+renewal (currently every 90 days at most)**. If interested in how to reduce
+this cost 😉, keep reading.
+
```
Comment (BirgitBader) [inline _posts/2019/2019-06-20-managing-lets-encrypt-certificates-in-vault.md]: Would love a headline for the following subsection here, as the following text block is quite long.

File: _posts/2019/2019-06-20-managing-lets-encrypt-certificates-in-vault.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+image](https://github.com/hashicorp/docker-vault/blob/2b7561b55940c35e412c914083e7dd40d21e9193/0.X/Dockerfile#L12-L46){:target="_blank"}
+with the removal of the `ca-certificates` package which is available from the
+base image and the addition of the `curl` and `jq` packages that we are going
+to need later on.
+
+Let's build this and tag it as `certbot-vault` by running the following on the
+same directory of our `Dockerfile`:
+
+```bash
+docker build -t certbot-vault .
```
Comment (BirgitBader) [inline _posts/2019/2019-06-20-managing-lets-encrypt-certificates-in-vault.md]: Is the dot correct here after certbot-vault?

**Comment by BirgitBader** [comment]
@ikoutras-epages Nicely written (although technically I don't understand a word 🤣 ) I did some clean up commits, and made two comments. Please have a look 😊

File: _posts/2019/2019-06-20-managing-lets-encrypt-certificates-in-vault.md (hunk 2 above)
Comment (ikoutras-epages) [reply _posts/2019/2019-06-20-managing-lets-encrypt-certificates-in-vault.md]: Yes, it is the directory into which Docker should look in order to find the Dockerfile.

File: _posts/2019/2019-06-20-managing-lets-encrypt-certificates-in-vault.md (hunk 1 above)
Comment (ikoutras-epages) [reply _posts/2019/2019-06-20-managing-lets-encrypt-certificates-in-vault.md]: Added one ("Automating certificate issuance") in 737bc2a.

**Comment by ikoutras-epages** [comment]
@BirgitBader thank you for your kind words, hope my post makes sense to the DevOps world :laughing:

Would it be OK to include this post in my personal blog? I don't have one yet, but in case I make one in the future, it is nice to know.

**Pull request merged**